20200429 11:47:28:387 [1] - System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 QuickFix.SocketInitiatorThread.ReadSome(Byte[] buffer, Int32 timeoutMilliseconds) 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 206
   在 QuickFix.SocketInitiatorThread.Read() 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 86
20200429 11:47:28:387 [1] -    在 QuickFix.SocketInitiatorThread.ReadSome(Byte[] buffer, Int32 timeoutMilliseconds) 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 206
   在 QuickFix.SocketInitiatorThread.Read() 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 86
20200429 11:48:28:054 [1] - System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 QuickFix.SocketInitiatorThread.ReadSome(Byte[] buffer, Int32 timeoutMilliseconds) 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 206
   在 QuickFix.SocketInitiatorThread.Read() 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 86
20200429 11:48:28:054 [1] -    在 QuickFix.SocketInitiatorThread.ReadSome(Byte[] buffer, Int32 timeoutMilliseconds) 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 206
   在 QuickFix.SocketInitiatorThread.Read() 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 86
20200429 11:49:28:262 [1] - System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 QuickFix.SocketInitiatorThread.ReadSome(Byte[] buffer, Int32 timeoutMilliseconds) 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 206
   在 QuickFix.SocketInitiatorThread.Read() 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 86
20200429 11:49:28:262 [1] -    在 QuickFix.SocketInitiatorThread.ReadSome(Byte[] buffer, Int32 timeoutMilliseconds) 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 206
   在 QuickFix.SocketInitiatorThread.Read() 位置 D:\ZProject\TTNew\TT V2\QuickFIXn\SocketInitiatorThread.cs:行号 86
